src/main/java/edu/harvard/iq/dataverse/DOIDataCiteRegisterService.java

```diff
@@ -144,8 +147,17 @@ private String getMetadataFromDvObject(String identifier, Map<String, String> me
         }
         metadataTemplate.setPublisher(producerString);
         metadataTemplate.setPublisherYear(metadata.get("datacite.publicationyear"));
-        String xmlMetadata = metadataTemplate.generateXML();
-        logger.log(Level.FINE, "XML to send to DataCite: {0}", xmlMetadata);
+        if (dvObject.isInstanceofDataFile()) {
```

Comment:
We do a check for isInstanceofDataFile() a few lines up, can we collapse these two bits of logic together?